.github/scripts/print_latest_commits.py

```diff
@@ -72,8 +63,14 @@ def get_commit_results(commit: str, results: Dict[str, Any]) -> List[Dict[str, A
 def isGreen(commit: str, results: Dict[str, Any]) -> Tuple[bool, str]:
     workflow_checks = get_commit_results(commit, results)
 
-    for required_check in regex:
-        regex[required_check] = False
+    regex = {
+        "^pull+": False,
```

Comment:
```suggestion
        "^pull.+": False,
```

Are the +'s signifying 1 or more? Cuz then wouldn't this regex be checking for pull(llllll...lll)? If so, how come you don't need a . for wildcard/how are your tests passing lol

Reply:
OH LOL so these all match for 

"pul(lll..lll)"
"trun(kk..kkk)"

etc.

A better regex would be just not having the + for each of these